user-guide/Troubleshooting/Known_issues/KI_SLProtocol_Crash_Enhanced_Services.md

```diff
@@ -0,0 +1,45 @@
+---
+uid: KI_SLProtocol_Crash_Enhanced_Services
+---
+
+# SLProtocol crashes when enhanced service protocol is missing table 100
+
+## Affected versions
+
+- All DataMiner versions
+
+## Cause
+
+The enhanced service alarm monitoring feature is not completely disabled when the enhanced service protocol is missing table 100. This can lead to access violations when alarm monitoring is enabled (PID 198) and elements are restarted.
+
+## Workaround
+
+1. Stop the DataMiner Agent.
+
+1. Run *Change Element States Offline.exe* as Administrator.
+
+   This tool is available on the DMA in the folder `C:\Skyline DataMiner\Tools` or `C:\Skyline DataMiner\Tools\Change Element States Offline`, depending on the DataMiner version. For more information , see [Change Element States Offline](xref:Change_Element_States_Offline).
+
+1. Go to the *Elements by Protocol* tab.
+
+1. Select all protocols.
+
+   > [!CAUTION]
+   > Older versions of this tool may also include **service protocols** in this list. These should **never be stopped** and should therefore be excluded from the mentioned selection. Service protocols have the type *service* in their *protocol.xml*.
+
+1. In the *Desired state for selected elements* dropdown box, select *Stopped*.
+
+1. Click the *SET STATE* button.
+
+1. Start the DataMiner Agent.
+
+1. In each enhanced service, set PID 198 to *disabled*, *0*, or *false* (depending on the protocol).
```

Comment:
Could you confirm if I interpreted this correctly in my review? The original sentence was a bit ambiguous.